keps/sig-node/135-seccomp/README.md

```diff
@@ -231,7 +231,7 @@ Cluster admins can build their own solutions to keep profiles in sync across nod
 [using daemonsets](https://gardener.cloud/050-tutorials/content/howto/secure-seccomp/)). Or use 
 community driven projects (e.g. [seccomp config](https://github.com/UKHomeOffice/seccomp-config), 
 [openshift's machine config operator](https://github.com/openshift/machine-config-operator), 
-[seccomp operator](https://github.com/saschagrunert/seccomp-operator)) that focuses on solving similar 
+[seccomp operator](https://github.com/kubernetes-sigs/security-profiles-operator)) that focuses on solving similar 
```

Comment:
```suggestion
[security-profiles-operator](https://github.com/kubernetes-sigs/security-profiles-operator)) that focuses on solving similar 
```

oops I missed this, should this be updated too?

Reply:
Yep, fixed that one too :+1: